build/mchn/DB/boot

```diff
@@ -2,6 +2,10 @@ set console wru=034
 set cpu its
 set cpu idle
 set tim y2k
+set ch enabled
+set ch node=177002
```

Comment:
node and peer should not be hardcoded, but should come from build-time options. If I have my own cbridge and subnet, these will not be good defaults. Just like we allow overriding of the IP address, we should allow overriding of the chaosnet address.

Reply:
You're not running DB ITS, but ES, EX, etc, right?